app/gitcoinbot/tests.py

```diff
@@ -81,21 +88,75 @@ def test_parse_tipee_username(self):
     def test_new_tip_text(self):
         """Test Gitcoinbot can respond with link to complete a tip."""
         issue_url = "https://github.com/test_owner/gitcoin/issues/1234"
-        tip_link = f"{settings.BASE_URL}tip/?amount=3.3&username=@user&source={issue_url}"
+        tip_link = f'{settings.BASE_URL}tip/?amount=3.3&tokenName=ETH&username=@user&source={issue_url}'
```

Comment:
Any reason for the switch from `'` to `"`?  Looks inconsistent with the rest of the method.

Reply:
yep, sorry